packages/test/test-drivers/src/localServerTestDriver.ts

```diff
@@ -22,14 +22,27 @@ export class LocalServerTestDriver implements ITestDriver {
     createDocumentServiceFactory(): IDocumentServiceFactory {
         return new this.api.LocalDocumentServiceFactory(this._server);
     }
+
     createUrlResolver(): IUrlResolver {
         return new this.api.LocalResolver();
     }
+
     createCreateNewRequest(testId: string): IRequest {
         return this.api.createLocalResolverCreateNewRequest(testId);
     }
 
     async createContainerUrl(testId: string): Promise<string> {
         return `http://localhost/${testId}`;
     }
+
+    async doesDocumentExists(testId: string): Promise<boolean> {
+        const doc = await (await this._server.databaseManager.getDocumentCollection()).findOne({
+            tenantId: "tenantId", // Its hardcoded for Local resolver.
```

Comment:
nit/suggestion (optional for this PR) -- is this coupled to `LocalResolver.tenantId` which is private and hardcoded to `"tenantId"`? Maybe make that public and reference it here for future-proofing?